src/lib.rs

```diff
@@ -26,14 +26,15 @@ use std::{collections::HashSet, pin::Pin};
 use arrayvec::ArrayVec;
 use libcec_sys::{
     cec_command, cec_datapacket, cec_device_type_list, cec_keypress, cec_log_message,
-    cec_logical_address, cec_logical_addresses, libcec_audio_get_status, libcec_audio_mute,
-    libcec_audio_toggle_mute, libcec_audio_unmute, libcec_clear_configuration, libcec_close,
-    libcec_configuration, libcec_connection_t, libcec_destroy, libcec_get_active_source,
-    libcec_get_device_power_status, libcec_initialise, libcec_is_active_source, libcec_mute_audio,
-    libcec_open, libcec_power_on_devices, libcec_send_key_release, libcec_send_keypress,
-    libcec_set_active_source, libcec_set_inactive_view, libcec_set_logical_address,
-    libcec_standby_devices, libcec_switch_monitoring, libcec_transmit, libcec_volume_down,
-    libcec_volume_up, ICECCallbacks, LIBCEC_OSD_NAME_SIZE, LIBCEC_VERSION_CURRENT,
+    cec_logical_address, cec_logical_addresses, cec_power_status, libcec_audio_get_status,
```

Comment:
The only change here is the addition of `cec_power_status`, but I thought it would be best to keep the alphabetical ordering. Add rustfmt, and this is the result 🤷